src/google/adk/tools/openapi_tool/openapi_spec_parser/tool_auth_handler.py

```diff
@@ -260,7 +255,7 @@ def prepare_auth_credentials(
           auth_credential=exchanged_credential,
       )
     else:
-      self._reqeust_credential()
+      self._request_credential()
```

Comment:
This is the only usage of `_reqeust_credential` in the library, and there are no usages of `_request_credential`. Given it is private, there should not be any public usages of this either.